### ISSUE TITLE
Missing Element definitions

### ISSUE THREAD

**Issue by almstrand** [nippur72/PolymerTS#44]
The following code runs as expected, though the TypeScript type checker does not recognize properties `clientWidth` or `clientHeight`:

```
/// <reference path="../../bower_components/polymer-ts/polymer-ts.ts" />
  @component("my-element") 
class MyElement extends polymer.Base {  
    attached() {
        console.log(this.clientWidth + 'x' + this.clientHeight); 
    }
} 
 MyElement.register();
```

Those properties are currently not defined in `polymer.Base`. Can this be corrected by having `polymer.Base` extend [`Element`](https://developer.mozilla.org/en-US/docs/Web/API/Element)?

Of course, this request assumes that all custom elements are indeed of type [`Element`](https://developer.mozilla.org/en-US/docs/Web/API/Element), and I am not familiar enough with the Polymer 1.0 implementation to know whether that is the case. Is it correct that custom elements are always created by calling [`createElement()`](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement) (which returns an object of type [`Element`](https://developer.mozilla.org/en-US/docs/Web/API/Element))?

I came across [this issue](https://github.com/Microsoft/TypeScript/issues/574) filed with the TypeScript GitHub repository, which seems relevant. Perhaps that issue needs to be addressed before a change can be made in PolymerTS?

**Comment by nippur72** [comment]
Yes, exactly, see also #31: 

*the wrong type is a temporary issue, we are waiting for [this issue](https://github.com/Microsoft/TypeScript/issues/574) to be solved in TypeScript. When it will be fixed, your "el" will be an extension of type `HTMLElement` and you will be able to do DOM operations with it.*

I tried once to copy `HTMLElement` from `lib.d.ts` but there's a very long chain, so we better wait it to be fixed (I hope in 1.6).

**Comment by nippur72** [comment]
@almstrand This issue is now fixed (#54) as TypeScript 1.6 has changed the way it declares `HTMLElement`.
